test/e2e/network/network_policy.go

```diff
@@ -98,6 +98,28 @@ var _ = SIGDescribe("NetworkPolicy [LinuxOnly]", func() {
 			testCannotConnect(f, f.Namespace, "client-cannot-connect", service, 80)
 		})
 
+		ginkgo.It("should support a 'default-deny-all' policy [Feature:NetworkPolicy]", func() {
+			policy := &networkingv1.NetworkPolicy{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "default-deny-all",
+				},
+				Spec: networkingv1.NetworkPolicySpec{
+					PodSelector: metav1.LabelSelector{},
+					PolicyTypes: []networkingv1.PolicyType{networkingv1.PolicyTypeEgress, networkingv1.PolicyTypeIngress},
+					Ingress:     []networkingv1.NetworkPolicyIngressRule{},
+					Egress:      []networkingv1.NetworkPolicyEgressRule{},
```

Comment:
I'm not familiar with these tests so please , but this is interesting 🤔 
 how do the tests know that the failure to connect is caused by the ingress or the egress?
`default-deny-all` and `default-deny-ingress` have the same behavior but I can't see how do we exercise each policy

Reply:
Thank you! You're totally right. this deny-all test should be more complex. I've updated PR, added pod in another namespace and check is it reachable after policy applied.